src/controller/testing.java

```diff
@@ -0,0 +1,5 @@
+package controller;
+
+public class testing {

```

Comment:
No sure whats the point of a empty testing class

Reply:
Only made it to have the folder appear on github. This was removed

Reply:
in later issue